src/darwin/CHIPTool/CHIPTool/Framework Helpers/DefaultsUtils.m

```diff
@@ -82,6 +82,20 @@ void CHIPSetNextAvailableDeviceID(uint64_t id)
     return device;
 }
 
+CHIPDevice * GetPairedDeviceWithID(uint64_t deviceId)
```

Comment:
Let's call this CHIPGetPairedDeviceWithID since all the methods here start that way